teammapper-frontend/package.json

```diff
@@ -73,15 +73,15 @@
     "zone.js": "^0.14.4"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "0.1703.2",
-    "@angular-devkit/core": "^17.3.2",
-    "@angular-devkit/schematics": "^17.3.2",
-    "@angular-eslint/builder": "17.3.0",
-    "@angular-eslint/eslint-plugin": "17.3.0",
-    "@angular-eslint/eslint-plugin-template": "17.3.0",
-    "@angular-eslint/schematics": "17.3.0",
-    "@angular-eslint/template-parser": "17.3.0",
-    "@angular/language-service": "^17.3.1",
+    "@angular-devkit/architect": "0.1801.2",
+    "@angular-devkit/core": "^18.1.2",
+    "@angular-devkit/schematics": "^18.1.2",
+    "@angular-eslint/builder": "18.1.0",
+    "@angular-eslint/eslint-plugin": "18.1.0",
+    "@angular-eslint/eslint-plugin-template": "18.1.0",
+    "@angular-eslint/schematics": "18.1.0",
+    "@angular-eslint/template-parser": "18.1.0",
+    "@angular/language-service": "^18.1.2",
     "@compodoc/compodoc": "^1.1.25",
     "@schematics/angular": "^17.3.2",
```

Comment:
is there also a v18 available?

Reply:
Yup, which will already be pulled in by @angular/cli, but I've updated the version to ^18.1.4 anyway.